### ISSUE TITLE
Ability to selected blueprints from a deactivated theme

### ISSUE THREAD

**Issue by drzraf** [getgrav/grav-plugin-admin#1983]
Transversal issue from https://github.com/getgrav/grav/issues/3082
tldr; I use [grav-plugin-themer](https://github.com/Sommerregen/grav-plugin-themer) which allow pages to be rendered, in the frontend, using another theme than the default one.

It does this by introducing a `themeselect` blueprint in `header.theme:` what works well for the frontend.
Sadly, it's not integrated with `admin` which obviously does not consider this YAML key in any specific way and just try to find the blueprint named after the name of the page. To make [per-page theme](https://github.com/getgrav/grav/issues/3082) possible, a change there would be needed so that the `admin` plugin could use another theme's blueprints to render the editing forms of a page.

**Comment by rhukster** [comment]
Themer is a 3rd party plugin, and as such would not have built-in support in the admin. However, it's possible that theme could make use of events to introduce functionality into the admin.  But as it's a 3rd party plugin, that would be the responsibility of the plugin author to investigate.

**Comment by drzraf** [comment]
The issue was opened ith getgrav/grav#3082 as both would provide a useful feature built into Grav instead of relying of the old/deprecated/partially working grav-plugin-themer (which was cited as an example to give some context).

As it's made clear, the feature itself (per-page theme) can't be implemented properly by a plugin alone. I opened this issue in the hope its core counterpart would gain traction (which would definitely need some tweaks from the admin-plugin then)

Could I suggest to leave it open in relation to getgrav/grav#3082?

**Comment by rhukster** [comment]
Frankly, I have no interest in developing this functionality.  It's just making things unnecessarily complex.  The admin needs to know what the current theme is because it impacts blueprints and templates available at the page level. Supporting this would be a huge effort, and potentially causing performance issues, not to mention opening things up to bugs and and other problems in pages that are already complex enough with support for multilang, custom blueprints, and all the other stuff.

All this for people that are running multiple themes with an unsupported third party plugin, when they could, with not too much effort, simply add differently styled pages to one single theme.

We have been burned too many times with requests for 'obscure' functionality, even with PRs, that have not been fully tested in the variety of situations that Grav can support, and ultimately causing breaking issues for other people with more common place use cases

I don't mean to be harsh, but we're really trying to get Grav to a nice stable and reliable state, not adding new functionality at this point.  We would rather be able to turn our focus to Grav 2.x which is going to be a major revamp.

**Comment by drzraf** [comment]
That makes perfect sense, backed by your global vision of the project.
Neither Drupal nor WP provide such a feature (Still I wouldn't call this an "obscure" because there are several valid real-life use-cases for this)

I hope the use-cases could be discussed and workarounds envisioned but is this issue the best place?

**Comment by mahagr** [comment]
Gantry provides customization within the theme and that may be something that could be added to Grav pretty easily (all it needs is some variables on the page to customize your looks).
